Template/Tag/CookieYesTag.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\TagManager\Template\Tag;
+
+use Piwik\Piwik;
+use Piwik\Settings\FieldConfig;
+use Piwik\Validators\NotEmpty;
+
+class CookieYesTag extends BaseTag
+{
+    public function getName()
+    {
+        // By default, the name will be automatically fetched from the TagManager_CustomHtmlTagName translation key.
+        // you can either adjust/create/remove this translation key, or return a different value here directly.
+        return parent::getName();
+    }
+
+    public function getDescription()
+    {
+        // By default, the description will be automatically fetched from the TagManager_CustomHtmlTagDescription
+        // translation key. you can either adjust/create/remove this translation key, or return a different value
+        // here directly.
+        return Piwik::translate('TagManager_CookieYesTagDescription');
+    }
+
+    public function getHelp()
+    {
+        // By default, the help will be automatically fetched from the TagManager_CustomHtmlTagHelp translation key.
+        // you can either adjust/create/remove this translation key, or return a different value here directly.
+        return parent::getHelp();
+    }
+
+    public function getCategory()
+    {
+        return Piwik::translate('TagManager_ConsentManagementPlatform');
+    }
+
+    public function getIcon()
+    {
+        // You may optionally specify a path to an image icon URL, for example:
+        //
+        // return 'plugins/TagManager/images/MyIcon.png';
+        //
+        // to not return default icon call:
+        // return parent::getIcon();
+        //
+        // The image should have ideally a resolution of about 64x64 pixels.
+        return 'plugins/TagManager/images/icons/cookieyes.svg';
+    }
+
+    public function getParameters()
+    {
+        return array(
+            $this->makeSetting('cookiebotWebsiteKey', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
```

Comment:
It doesn't need to be changed, but I wonder if this is a copy/paste from another tag and not updated? It might be easier just to leave it rather than change all the other references. I worry a little that it might cause confusion, but it will likely be fine.
```suggestion
            $this->makeSetting('cookieYesWebsiteKey', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
```